src/features/validation/ComponentValidations.tsx

```diff
@@ -50,42 +50,45 @@ export function ComponentValidations({ validations, node: _node }: Props) {
   const info = validationsOfSeverity(filteredValidations, 'info');
   const success = validationsOfSeverity(filteredValidations, 'success');
 
+  if (!node || !validations?.length) {
+    // eslint-disable-next-line react/jsx-no-useless-fragment
+    return <></>;
```

Comment:
Why is the useless fragment needed? :sweat_smile:
```suggestion
    return null;
```